client/src/redux/reducers/projects.reducers.js

```diff
@@ -3,7 +3,52 @@ export const initialState = {
     projects: [],
     error: null,
     loading: false
+  },
+  operation: {
+    name: '',
+    loading: false,
+    error: null,
+    result: null
+  },
+  metadata: {
+    title: '',
+    description: '',
+    projectId: '',
+    members: [],
+    loading: false,
+    error: null
+  }
+};
+
+const addProjectMember = (members, newMember) => {
+  return [...members, newMember];
+};
+
+const removeProjectMember = (members, removedMember) => {
+  const index = members.findIndex(
+    el => el.user.username === removedMember.user.username
+  );
+  if (index) {
+    members.splice(index, 1);
   }
+  return [...members];
```

Comment:
Think this could be a one liner
```suggestion
  return members.filter(el => el.user.username !== removedMember.user.username)
```